apps/server/src/modules/learnroom/controller/rooms.controller.spec.ts

```diff
@@ -0,0 +1,85 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { EntityId, ICurrentUser } from '@shared/domain';
+import { BoardMapper } from '../mapper/board.mapper';
+import { RoomsUc, Board } from '../uc/rooms.uc';
+import { BoardResponse } from './dto/roomBoardResponse';
+import { RoomsController } from './rooms.controller';
+
+describe('rooms controller', () => {
+	let controller: RoomsController;
+	let mapper: BoardMapper;
+	let uc: RoomsUc;
+
+	beforeEach(async () => {
+		const module: TestingModule = await Test.createTestingModule({
+			imports: [],
+			providers: [
+				RoomsController,
+				{
+					provide: RoomsUc,
+					useValue: {
+						// eslint-disable-next-line @typescript-eslint/no-unused-vars
+						getBoard(roomId: EntityId, userId: EntityId): Promise<Board> {
+							throw new Error('please write mock for RoomsUc.getBoard');
+						},
+					},
+				},
+				{
+					provide: BoardMapper,
+					useValue: {
+						// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
comment still needed?

Reply:
absolutely, otherwise the unused vars would cause linter warnings.